cli/src/lib.rs

```diff
@@ -319,7 +319,7 @@ impl Iroha {
             let wsv_clone = wsv.clone();
 
             transaction_validator
-                .validate_every(genesis.iter().cloned(), &wsv_clone)
+                .validate_every::<true>(genesis.iter().cloned(), &wsv_clone)
```

Comment:
I'm not at all a fan of this API. I'd rather we had a similarly named method in two different traits. The user is left guessing as to what `<true>` means in this context. It was slightly better when we had an enum, but now I'm convinced we really need two trait extension methods and only one in scope.